tutorial.css

```diff
@@ -80,9 +80,9 @@ in the center, so it's placed to the left. These are used with .knit_border and
 
 #exercise {
   display: block;
-  width: 98%;
+  width: 100%;
```

Comment:
Just tried building the website with quilt(). 

Apparently this works. I wonder why it didn't before